app/scripts/services/rest.js

```diff
@@ -34,8 +34,9 @@ angular.module('openhimWebui2App')
       Mediators: $resource( server + '/mediators/:urn', { urn: '@urn' }),
 
       // add the metric endpoints
-      Metrics: $resource( server + '/metrics/:type/:channelId', {}),
-      MetricsStatus: $resource( server + '/metrics/status', {}),
+      Metrics: $resource( server + '/stats/:type/:channelId', {}),
```

Comment:
Hasn't the endpoint been changed back to `/metrics/` on the statsd model branch?